diff -rupN mm-org/build/release.mk mm-mod/build/release.mk
--- mm-org/build/release.mk	2023-01-30 12:51:25.527052880 +0100
+++ mm-mod/build/release.mk	2023-01-30 12:52:32.043053709 +0100
@@ -200,9 +200,6 @@ endif
 		echo "Copying web app files for Boards product"; \
 		mkdir -p $(DIST_PATH_GENERIC)/client/products/boards; \
 		cp -R $(BUILD_BOARDS_DIR)/mattermost-plugin/webapp/dist/* $(DIST_PATH_GENERIC)/client/products/boards/; \
-	else \
-		echo "Unable to find files for Boards product. Please ensure that the Focalboard repository is checked out alongside the server and run 'make build-product' in it."; \
-		exit 1; \
 	fi
 
 package-osx-amd64: package-prep
